scripts/prepare_network.py

```diff
@@ -372,10 +373,10 @@ def set_line_nom_max(n, s_nom_max_set=np.inf, p_nom_max_set=np.inf):
                     co2limit = co2limit * float(m[0])
                 logger.info("Setting CO2 limit according to emission base year.")
             elif len(m) > 0:
-                co2limit = float(m[0]) * snakemake.params.electricity["co2base"]
+                co2limit = float(m[0]) * float(snakemake.params.electricity["co2base"])
                 logger.info("Setting CO2 limit according to wildcard value.")
             else:
-                co2limit = snakemake.params.electricity["co2limit"]
+                co2limit = float(snakemake.params.electricity["co2limit"])
```

Comment:
My guess is that these changes in `prepare_network` have been needed to test the workflow properly. Can you please confirm that? 🙂 (apologies for an idiotic question, just double-checking that the changes are intentional and exactly as they are supposed to be)

Reply:
Today there has been a temporary issue with the automatic download of emission data; because of that I disabled that and I had issues as the float(..) was needed to make it work.
It was a minor thing and decided to add it as it is minor